diff --git a/node_modules/resolve-url-loader/index.js b/node_modules/resolve-url-loader/index.js
index b4e365e..b34d37d 100644
--- a/node_modules/resolve-url-loader/index.js
+++ b/node_modules/resolve-url-loader/index.js
@@ -50,7 +50,7 @@ function resolveUrlLoader(content, sourceMap) {
       silent   : false,
       absolute : false,
       keepQuery: false,
-      removeCR : false,
+      removeCR : true,
       root     : false,
       debug    : false,
       join     : joinFn.defaultJoin
